custom/src/main/java/com/splunk/opentelemetry/SplunkTracerCustomizer.java

```diff
@@ -42,5 +43,9 @@ public void configure(TracerSdkManagement tracerManagement) {
     if (jdbcSpanLowCardinalityNameEnabled()) {
       tracerManagement.addSpanProcessor(new JdbcSpanRenamingProcessor());
     }
+
+    tracerManagement.updateActiveTraceConfig(tracerManagement.getActiveTraceConfig().toBuilder()
+            .setSampler(Sampler.alwaysOn())
```

Comment:
The SDK itself defaults to alwaysOn. Is the otel agent doing something different?

Reply:
The SDK actually defaults to

```java
  private static final Sampler DEFAULT_SAMPLER = Sampler.parentBased(Sampler.alwaysOn());
```

which works slightly different: for spans whose parent is not sampled it also returns the "not sampled" flag

Reply:
aha! You are totally correct. I conflated the two in my head.